packages/runtime/native/shared/Css_types.ml

```diff
@@ -4064,6 +4064,23 @@ module FontVariantEmoji = struct
     | #Cascading.t as c -> Cascading.toString c
 end
 
+module URange = struct
+  type t =
+    [ `single of string
```

Comment:
I would turn strings into ints, right?

Reply:
My mistake, doing it.

Reply:
Just tried it but it breaks the hex string_of_int:

E.g: 0x3F Becomes 63.

I can make it work with Printf.sprintf, but it breaks the CI on rescript:

```
  We've found a bug for you!
  packages/runtime/native/shared/Css_types.ml:4079:28-41

  The module or file Printf can't be found.
  - If it's a third-party dependency:
    - Did you list it in bsconfig.json?
    - Did you run `rescript build` instead of `rescript build -with-deps`
      (latter builds third-parties)?
  - Did you include the file's directory in bsconfig.json?
  ```
Do you have an alternative in mind?  Can It be added to dependency? Or it should be created a function to handle this string_of_hex_int?

Reply:
oh, crap. Let's keep using strings for this.